README.rst

```diff
@@ -258,8 +258,28 @@ existing ones. These are know to work well with Volto layout and grid system::
 sure your add-on's profiles are applied AFTER this one.**
 
 
+Queryparser operation for ObjectBrowser
+---------------------------------------
+
+There is a new operation "*plone.app.querystring.operation.string.objectBrowserReference*" for path index in querystring.
```

Comment:
```suggestion
There is a new operation `plone.app.querystring.operation.string.objectBrowserReference` for path index in `querystring`.
```

Reply:
I would skip "new", since it is old/normal soon.